### PR TITLE
optimize: optimize channel availability thread log output

### PR DESCRIPTION
**Pull request by tobehardest**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
The stack is output under debug, and the stack is not output under info, and all failed addresses are combined and output instead of 5 addresses out of 10 being unavailable and outputting logs five times.

### Ⅱ. Does this pull request fix one issue?

fiexs: #5474 

### Ⅳ. Describe how to verify it
view log

### Ⅴ. Special notes for reviews
None

### CODE REVIEW

**Review by slievrly — COMMENTED** [review]
any. file changed?

**Comment by tobehardest** [comment]
> any. file changed?
My fault, it has been changed.

File: core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java [hunk 1]
```diff
@@ -190,14 +190,26 @@ void reconnect(String transactionServiceGroup) {
             return;
         }
         Set<String> channelAddress = new HashSet<>(availList.size());
+        List<String> failedList = new ArrayList<>(), exceptionMessageList = new ArrayList<>();
```
Comment (a364176773) [inline core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java:193]: why not k,v structure

File: core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java [hunk 2]
```diff
@@ -190,14 +190,26 @@ void reconnect(String transactionServiceGroup) {
...
+        List<Exception> exceptionList = new ArrayList<>();
```
Comment (a364176773) [inline core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java:194]: exception itself contains the message

Comment (tobehardest) [reply core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java]: > exception itself contains the message

thank you. I redid it

File: core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java [hunk 3]
```diff
@@ -190,14 +192,23 @@ void reconnect(String transactionServiceGroup) {
...
                     channelAddress.add(serverAddress);
                 } catch (Exception e) {
+                    failedMap.put(serverAddress, e);
+                }
+            }
+            if (failedMap.size() > 0) {
+                if (LOGGER.isInfoEnabled()) {
                     LOGGER.error("{} can not connect to {} cause:{}", FrameworkErrorCode.NetConnect.getErrCode(),
-                        serverAddress, e.getMessage(), e);
+                            failedMap.keySet(), failedMap.values().stream().map(Throwable::getMessage).collect(Collectors.toList()));
```
Comment (a364176773) [inline core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java:208]: 应该toSet这样重复的msg应该会被去重，避免无意义的提示

File: core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java [hunk 4]
```diff
@@ -190,14 +192,23 @@ void reconnect(String transactionServiceGroup) {
...
+                } else if (LOGGER.isDebugEnabled()) {
+                    LOGGER.error("{} can not connect to {} cause:{} trace information:{}", FrameworkErrorCode.NetConnect.getErrCode(),
```
Comment (a364176773) [inline core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java:210]: debug我建议就不要做合并处理了按原逻辑处理，直接foreach 每个address都输出一遍吧

**Comment by tobehardest** [comment]
Updated.

**Review by slievrly — APPROVED** [review]
LGTM , Pls add changes in:
https://github.com/seata/seata/blob/2.x/changes/en-us/2.0.0.md
https://github.com/seata/seata/blob/2.x/changes/zh-cn/2.0.0.md

**Pull request merged**